web/components/manalinks/create-links-button.tsx

```diff
@@ -87,22 +115,27 @@ function CreateManalinkForm(props: {
           <div className="flex flex-col flex-wrap gap-x-5 gap-y-2">
             <div className="form-control flex-auto">
               <label className="label">Amount</label>
-              <input
-                className="input input-bordered"
-                type="number"
-                value={newManalink.amount}
-                onChange={(e) =>
-                  setNewManalink((m) => {
-                    return { ...m, amount: parseInt(e.target.value) }
-                  })
-                }
-              ></input>
+              <div className="relative">
+                <span className="absolute mx-3 mt-3.5 text-sm text-gray-400">
+                  M$
+                </span>
+                <input
+                  className="input input-bordered w-full pl-10"
+                  type="number"
+                  value={newManalink.amount}
+                  onChange={(e) =>
+                    setNewManalink((m) => {
+                      return { ...m, amount: parseInt(e.target.value) }
+                    })
+                  }
+                />
+              </div>
             </div>
             <div className="flex flex-col gap-2 md:flex-row">
-              <div className="form-control">
+              <div className="form-control w-full md:w-1/2">
                 <label className="label">Uses</label>
                 <input
-                  className="input input-bordered w-full"
+                  className="input input-bordered"
                   type="number"
                   value={newManalink.maxUses ?? ''}
                   onChange={(e) =>
```

Comment:
Hm, we probably shouldn't be able to set negative maxUses: 
![image](https://user-images.githubusercontent.com/856034/179033491-5c7a22a0-8c5f-4717-b10b-c9575216a0b6.png)

Reply:
Set both amount and uses to have a minimum of 1